build-aux/snap/local/patch-dl.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
```

Comment:
it would be good for this to have a comment section to have a section explaining what it does